lxd-user/lxd.go

```diff
@@ -169,8 +169,6 @@ func lxdSetupUser(uid uint32) error {
 		return fmt.Errorf("Unable to connect to LXD: %w", err)
 	}
 
-	client.GetServer()
```

Comment:
Why did this go? I mean it doesn't look like its doing anything, but wonder why it was there in the first place (perhaps to trigger a connection?).

Reply:
CC @stgraber

Reply:
I've re-added this and explicitly ignored it's returned values just in case. If it is necessary we should add a comment to say why it is there.